fe/fe-core/src/main/java/org/apache/doris/planner/DistributedPlanner.java

```diff
@@ -375,6 +375,15 @@ private PlanFragment createHashJoinFragment(HashJoinNode node, PlanFragment righ
             doBroadcast = false;
         }
 
+        // Push down the predicates constructed by the right child when the
```

Comment:
It feels weird to set a value here uniformly, and then change it later.

Reply:
ok，I will rewrite this code.